crates/optimism/node/src/args.rs

```diff
@@ -55,6 +55,10 @@ pub struct RollupArgs {
         default_value_t = SafetyLevel::CrossUnsafe,
     )]
     pub supervisor_safety_level: SafetyLevel,
+
+    /// Optional headers to use when connecting to the sequencer.
+    #[arg(long = "rollup.sequencer-headers")]
+    pub sequencer_headers: Option<Vec<String>>,
```

Comment:
we can move this below the `sequencer` field

and this can jsut be Vec String

this must also be marked with `requires = "sequencer"`